Formula/a/autocycler.rb

```diff
@@ -0,0 +1,26 @@
+class Autocycler < Formula
+  desc "Tool for generating consensus long-read assemblies for bacterial genomes"
+  homepage "https://github.com/rrwick/Autocycler"
+  url "https://github.com/rrwick/Autocycler/archive/refs/tags/v0.4.0.tar.gz"
+  sha256 "028586ba433ddb6a21e4159c5e9075423c2ac4b0c60fbaa4744358cdfebf7d57"
+  license "GPL-3.0-or-later"
```

Comment:
any reason of not including a head build?

Reply:
No, will start doing this from now on

Reply:
Yeah, not all formulae support head builds, but Rust-based formulae typically handle them well.